pkg/apis/core/types.go

```diff
@@ -3761,6 +3761,27 @@ type PodResourceClaimStatus struct {
 	ResourceClaimName *string
 }
 
+// PodExtendedResourceClaimStatus is stored in the PodStatus for the extended
+// resource requests backed by DRA. It stores the generated name for
+// the corresponding special ResourceClaim created by kube-scheduler.
+type PodExtendedResourceClaimStatus struct {
+	// RequestMapping identifies the mapping of <container, extended resource backed by DRA> to  device request.
+	RequestMapping []ContainerExtendedResourceRequest
```

Comment:
The size of this is limited by the number of containers per pod and resource requests in each, so no extra limit necessary. Validation must check the keys.

Reply:
validatePodExtendedResourceClaimStatus() currently validates all the fields, is that sufficient?

Reply:
added the validation to check duplicate  {containerName,extendedResourceName} items in the requestMapping